src/Bicep.Core/Emit/TemplateWriter.cs

```diff
@@ -1450,16 +1403,6 @@ private void EmitModuleExtensionConfigs(ExpressionEmitter emitter, DeclaredModul
                                                     // write a single property copy loop
                                                     emitter.EmitObjectProperty(extConfigPropertyName, () => { emitter.EmitCopyProperty(() => { emitter.EmitArray(() => { emitter.EmitCopyObject("value", @for.Expression, @for.Body, "value"); }, @for.SourceSyntax); }); });
                                                 }
-                                                else if (extConfigPropertyExpr.Value is ResourceReferenceExpression resource &&
```

Comment:
@kalbert312 I removed this because `ResourceParameterType` isn't currently supported by `Bicep.Types`. We can reintroduce it later, but it will require some design work first.